src/elements/content-sidebar/activity-feed/task-new/Task.js

```diff
@@ -288,6 +288,7 @@ class Task extends React.Component<Props, State> {
                                     {permissions.can_update && (
                                         <MenuItem
                                             data-resin-target={ACTIVITY_TARGETS.INLINE_EDIT}
+                                            data-resin-component="tasks"
```

Comment:
Does resin work for menu items? I recall the consuming code looking for specific element types, like anchors and buttons. If it does work, can this property live on `Media.Menu` so it's not duplicated and will propagate to any actions we add going forward?

Reply:
@booc0mtaco @alexkrolick what do you think?

Reply:
The reason we are adding it here is because adding the attribute onto the Media.Menu or even the wrapped Tether component would pass the prop onto an element that isn't inside the Portal. Resin's event bubbling follows the DOM, not React's synthetic event propagation.

Reply:
Ah, gotcha. I implemented something similar for version history actions, but I'm using `Menu` directly there. I also double-checked the resin library and it supports selecting components by role as well as tag, so you should be good to go.